### ISSUE TITLE
C++ executable not starting in Windows docker

### ISSUE THREAD

**Issue by anasokan** [docker/for-win#700]
### Expected behavior
The C++ binary not starting

### Actual behavior
The C++ binary should start
### Information

I am using Windows 10 docker microsoft/windowsservercore.
I am executing my executable made of C++ and it is working on the actual Windows 10 host.
But on the docker of type Windows 10 created inside that machine its not working.
The strange part is no error is throw or no print is given to detect whats wrong.

How to detect what is the cause of the error

### Steps to reproduce the behavior
NA

**Comment by arasheedu** [comment]
I am able to reproduce this issue.

I built a simple console application using VS2017 that just prints hello world to the output. I tried both x86 and x64 builds. Neither of the print the console output when I am running the WindowsServerCore image interactively running CMD.exe.

However when I looked at the dependencies of my console application they seem to depend on the below dlls:

![image](https://user-images.githubusercontent.com/8011931/27095110-6f14bed6-5021-11e7-85ad-b5c3cc14f6ab.png)

When I copied these DLLs from C:\Windows\System32 folder along with the EXE inside the container it seems to print the console output.

So the basic problem is that dependency DLLs are missing. Not sure if this is a WindowsServerCore image issue whereby it is missing these DLLs that exist on C:\Windows\System32 folder on typical system.

Thanks!
